## queries where the current trajectory is along its path

# if trajectoryid is 0, returns the timestamp of the current trajectory
# and done is filled whenever all the trajectories have been finished
int32 trajectory_id
int32 Query_Joint_Names=-1
---
uint8 State_Active=0
uint8 State_Done=1
uint8 State_Queued=2
uint8 State_Deleted=3
uint8 State_Failed=4
uint8 State_Canceled=5
uint8 State_Does_Not_Exist=5

uint8 trajectory_status # 1 if trajectory is done, 0 if ongoing, 2 if queued, 3 if deleted, 4 if failed, 5 if canceled

float32 trajectory_time # the current timestamp the trajectory is at. If the trajectory is finished, specifies the total time of the trajectory

string[] joint_names # names of the joints that the controller expects
float64[] joint_positions # joint values
